hawc/apps/common/helper.py

```diff
@@ -411,3 +413,23 @@ def event_plot(series: pd.Series) -> Axes:
 
     plt.tight_layout()
     return ax
+
+
+class PydanticToDjangoError:
+    def __init__(self, include_field=True, field=None, msg=None, drf=False):
```

Comment:
add some type annotations and docstring?

Reply:
also, if drf is True, does it need to throw a `rest_framework.exceptions.ValidationError` instead of a django one?  I'm not 100% sure, but they dont have the same exception inheritance

Reply:
DRF is pretty good about handling Django ValidationErrors, but its a better idea to explicitly throw the correct error; this has been added!